### DISCUSSION TITLE
Google Assistant for reSpeaker | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#64] [Q&A]
# Google Assistant for reSpeaker | Seeed Studio Wiki

Google Assistant for reSpeaker

https://wiki.seeedstudio.com/Google_Assistant/

**Comment by AndresPacheco24** [comment]
Please help, in the Respeakerd installation process when I execute: 
  sudo python setup.py install 
I get this error: 
  Searching for pydbus<1,>=0.6.0
  Reading https://pypi.python.org/simple/pydbus/
  Couldn't find index page for 'pydbus' (maybe misspelled?)
  Scanning index of all packages (this may take a while)
  Reading https://pypi.python.org/simple/
  No local packages or working download links found for pydbus<1,>=0.6.0
  error: Could not find suitable distribution for Requirement.parse('pydbus<1,>=0.6.0')

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
